src/mjolnir/valhalla_build_connectivity.cc

```diff
@@ -14,15 +14,13 @@ using namespace valhalla::baldr;
 
 #include "baldr/rapidjson_utils.h"
 #include <boost/optional.hpp>
-#include <boost/program_options.hpp>
 #include <boost/property_tree/ptree.hpp>
+#include <cxxopts.hpp>
 #include <ostream>
 
-namespace bpo = boost::program_options;
 using namespace valhalla::midgard;
 
 filesystem::path config_file_path;
-std::vector<std::string> input_files;
```

Comment:
there's not input files here. you'll see that in a few exes, guess a copy/paste fail